docs/csharp/language-reference/operators/conditional-operator.md

```diff
@@ -10,20 +10,37 @@ helpviewer_keywords:
   - "conditional operator (?:) [C#]"
 ms.assetid: e83a17f1-7500-48ba-8bee-2fbc4c847af4
 ---
+
 # ?: Operator (C# Reference)
+
 The conditional operator (`?:`), commonly known as the ternary conditional operator, returns one of two values depending on the value of a Boolean expression. Following is the syntax for the conditional operator.  
-  
-```  
+
+```csharp
 condition ? first_expression : second_expression;  
-```  
-  
-## Remarks  
- The `condition` must evaluate to `true` or `false`. If `condition` is `true`, `first_expression` is evaluated and becomes the result. If `condition` is `false`, `second_expression` is evaluated and becomes the result. Only one of the two expressions is evaluated.  
-  
- Either the type of `first_expression` and `second_expression` must be the same, or an implicit conversion must exist from one type to the other.  
-  
- You can express calculations that might otherwise require an `if-else` construction more concisely by using the conditional operator. For example, the following code uses first an `if` statement and then a conditional operator to classify an integer as positive or negative.  
-  
+```
+
+Beginning with C# 7.2, the `first_expression` and `second_expression` my be `ref` expressions:
```

Comment:
It would be helpful to make ref expressions a link to the C# topic.

Reply:
I haven't been doing that because the spec we host is the C# 6 spec. Should I link to the design proposal in the dotnet/csharplang repo? 

What are your thoughts?

Reply:
I think that's fine.